arviz/stats/stats.py

```diff
@@ -885,67 +834,20 @@ def summary(
         summary_df = df
     else:
         summary_df = joined
```

Comment:
Here joined is still an xarray object. It should probably be `joined.to_dataframe()`.

Reply:
I think this is on purpose

Reply:
But then it will raise an error afterwards when calling `summary_df.columns`. pandas Dataframes have this attribute, but xarray objects do not.

Reply:
Yes, this is true, I fixed this problem by returning not-rounded version for xarray.